### PR TITLE
(Fix && Refactoring) Infura usage if no Metamask to get crowdsale data

### PR DESCRIPTION
**Pull request by vbaranov**
Closes https://github.com/poanetwork/token-wizard/issues/755

**Problem**:  users are not able to see crowdsale data at invest page on mobile devices
**Solution**: connect to Infura, if Metamask is not present to display data in crowdsale, invest pages.

Also, refactoring of `getTokenData` function is implemented

### CODE REVIEW

**Comment by vbaranov** [comment]
☝️minor fixes: unused action, setProperty, duplicated setState are removed

**Comment by vbaranov** [comment]
@fvictorio will you have a chance to review this PR?

**Comment by fvictorio** [comment]
Yes, sorry, I'll review it ASAP.

File: src/stores/Web3Store.js [hunk 1]
```diff
@@ -19,25 +22,55 @@ class Web3Store {
     })
   }
 
+  getInfuraLink = (network) => {
+    const infuraToken = 'kEpzZR9fIyO3a8gTqJcI'
+    return `https://${network}.infura.io/${infuraToken}`
```
Comment (fvictorio) [inline src/stores/Web3Store.js]: Maybe this should be obtained from an environment variable?

**Comment by fvictorio** [comment]
Just a comment. Otherwise, LGTM.

File: src/stores/Web3Store.js (hunk 1 above)
Comment (vbaranov) [reply src/stores/Web3Store.js]: Thanks, will add.

**Pull request merged**